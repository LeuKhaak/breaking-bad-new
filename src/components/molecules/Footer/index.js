import React from "react";
import * as styles from "./styles.module.scss";

function Footer() {
  const date = new Date();
  const year = date.getFullYear();
  return (
    <footer className={styles.footer}>
      <div className={styles.container}>
        <div>{year}</div>
        <div>@autor</div>
      </div>
    </footer>
  );
}

export default Footer;
