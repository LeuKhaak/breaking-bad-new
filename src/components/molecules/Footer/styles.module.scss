@import "../../../assets/styles/styles.scss";
.footer {
  .container {
    @include content;
    max-width: 74.5rem;
    display: flex;
    justify-content: space-between;
    padding: 1.75rem 0;
  }
}
