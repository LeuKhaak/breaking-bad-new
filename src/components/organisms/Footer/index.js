import React from "react";
import * as styles from "./styles.module.scss";

function Footer() {
  return (
    <footer className={styles.footer}>
      <div className={styles.container}>
        <div className={styles.footerText}>{new Date().getFullYear()}</div>
        <div className={styles.footerText}>@autor</div>
      </div>
    </footer>
  );
}

export default Footer;
