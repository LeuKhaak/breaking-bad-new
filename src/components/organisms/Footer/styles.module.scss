@import "../../../assets/styles/styles.scss";
.footer {
  .container {
    @include content;
    max-width: 74.5rem;
    display: flex;
    justify-content: space-between;
    padding: 1.75rem 0;
    .footerText {
      height: 1.5rem;
      display: flex;
      align-items: center;
    }
  }
}
