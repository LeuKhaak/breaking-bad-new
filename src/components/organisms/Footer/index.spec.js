import React from "react";
import { shallow } from "enzyme";
import Footer from ".";

let component = null;
const setComponent = () => shallow(<Footer />);
beforeEach(() => {
  component = setComponent();
});

describe("Footer testing:", () => {
  test("is Footer rendered", () => {
    component = setComponent();
    expect(component).toMatchSnapshot();
  });
});
