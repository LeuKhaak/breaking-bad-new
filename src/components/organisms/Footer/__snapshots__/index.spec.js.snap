// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing: is Footer rendered 1`] = `
<footer>
  <div>
    <div>
      2021
    </div>
    <div>
      @autor
    </div>
  </div>
</footer>
`;
